tests/foreman/cli/test_discoveredhost.py

```diff
@@ -271,120 +205,6 @@ def test_positive_provision_pxeless_bios_syslinux(self):
-        loader
-
-        :id: 0704ec80-dfaf-4f25-ad6b-a4cd3f33a1cb
-
-        :Setup:
-            1. Craft the FDI with remaster the image to have ssh enabled
-            2. Synchronize RHEL6 repo (needed for GRUB)
-
-        :Steps:
-            1. Create a UEFI VM and set it to boot from the FDI
-            2. Run assertion steps #1-2
-            3. Provision the discovered host using UEFI_GRUB PXE loader
-            4. Run assertion steps #3-7
-
-        :expectedresults: Host should be provisioned successfully
-            1. [TBD] Ensure FDI loaded and successfully sent out facts
-
-                1.1 ping vm
-                1.2 ssh to the VM and read the logs (if ssh enabled)
-                1.3 optionally sniff the HTTP traffic coming from the host
-
-            2. Ensure host appeared in Discovered Hosts on satellite
-            3. [TBD] Ensure the kexec was successful (e.g. the kexec request
-               result in production.log)
-            4. [TBD] Ensure anaconda loaded and the installation finished
-            5. [TBD] Ensure the host is provisioned with correct attributes
-            6. Ensure the host is created in Hosts
-            7. Ensure the entry from discovered host list disappeared
-
-        :CaseAutomation: NotAutomated
-
-        :CaseImportance: High
-        """
-
-    @pytest.mark.stubbed
-    @pytest.mark.tier3
-    def test_positive_provision_pxeless_uefi_grub2(self):
-        """Provision and discover the pxe-less UEFI host from cli using GRUB2
-        loader
-
-        :id: 81b0b586-617c-4100-80e6-ce924fb26d88
-
-        :Setup:
-            1. Craft the FDI with remaster the image to have ssh enabled
-            2. Synchronize RHEL7+ repo (needed for GRUB2)
-
-        :Steps:
-            1. Create a UEFI VM and set it to boot from the FDI
-            2. Run assertion steps #1-2
-            3. Provision the discovered host using GRUB2_UEFI PXE loader
-            4. Run assertion steps #3-7
-
-        :expectedresults: Host should be provisioned successfully
-            1. Ensure FDI loaded and successfully sent out facts
-
-                1.1 ping vm
-                1.2 ssh to the VM and read the logs (if ssh enabled)
-                1.3 optionally sniff the HTTP traffic coming from the host
-
-            2. Ensure host appeared in Discovered Hosts on satellite
-            3. Ensure the kexec was successful (e.g. the kexec request
-               result in production.log)
-            4. Ensure anaconda loaded and the installation finished
-            5. Ensure the host is provisioned with correct attributes
-            6. Ensure the host is created in Hosts
-            7. Ensure the entry from discovered host list disappeared
-
-        :CaseAutomation: NotAutomated
-
-        :CaseImportance: High
-        """
-
-    @pytest.mark.stubbed
-    @pytest.mark.tier3
-    def test_positive_provision_pxeless_uefi_grub2_secureboot(self):
```

Comment:
Secureboot is not supported, also the firwmare doesn't matter in pxeless scenario